### PR TITLE
Dtspo 8028 trafik v2 mig sboxptl

### PR DESCRIPTION
**Pull request by nickmoyni**
## JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/DTSPO-8028

### Change description ###

Added treafik-v2 configuration to SBOXPTL or SBOX-INTSVC 

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: apps/admin/sbox-intsvc/base/kustomization.yaml [hunk 1]
```diff
@@ -3,7 +3,12 @@ kind: Kustomization
...
-  - traefik-values.yaml
   - ../../../rbac/reader-clusterrole.yaml
-  - ../../traefik/traefik.yaml
+  - ../../secrets-csi-driver-crds
+  - ../../secrets-csi-driver
+  - ../../traefik2
+  - ../../aad-pod-id/mi
+
+patchesStrategicMerge:
+  - ../../traefik2/sbox/secret-provider.yaml
```
Comment (thomast1906) [inline apps/admin/sbox-intsvc/base/kustomization.yaml:13]: This isn't correct?

File: apps/admin/sbox-intsvc/base/kustomization.yaml [hunk 2]
```diff
@@ -3,7 +3,12 @@ kind: Kustomization
...
+  - ../../aad-pod-id/mi/sbox/azure-identity-patch.yaml
```
Comment (thomast1906) [inline apps/admin/sbox-intsvc/base/kustomization.yaml:14]: it won't be hte sbox MI

File: apps/admin/sbox-intsvc/base/kustomization.yaml (hunk 1 above)
Comment (nickmoyni) [reply apps/admin/sbox-intsvc/base/kustomization.yaml]: Corrected this, added own sbox-intsvc secret provider file and granted access PTLSBOX MI to that access policy.

File: apps/admin/sbox-intsvc/base/kustomization.yaml (hunk 2 above)
Comment (nickmoyni) [reply apps/admin/sbox-intsvc/base/kustomization.yaml]: Added own sbox-intsvc folder.

**Review by thomast1906 — APPROVED** [review]

**Pull request merged**